### PR TITLE
Drop support for python 2

### PR DESCRIPTION
**Pull request by kmcelwee**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,8 +46,6 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```
Comment (rlskoeser) [inline setup.py:51]: Looks like I forgot to add py3.7 here when I added it to the test matrix, would you add it?

File: .travis.yml [hunk 2]
```diff
@@ -17,15 +16,6 @@ env:
   - DJANGO=2.1
   - DJANGO=2.2
```
Comment (rlskoeser) [inline .travis.yml:17]: Would be nice to try newer versions of django and see whether or not we're compatible, but I guess that probably shouldn't be done right now.

File: README.rst [hunk 3]
```diff
@@ -30,9 +30,10 @@ support for prepopulating user account data based on an LDAP search.
 **django-pucas** is tested under:
 
 * Django ``1.8-2.2``
-* Python ``2.7, 3.5-3.7`` (excluding ``2.7`` for Django ``2+``)
+* Python ``3.5-3.7``
 
 **django-pucas** requires **django-cas-ng** 3.6 or greater.
+Support for Python 2 has been dropped.
```
Comment (rlskoeser) [inline README.rst]: This doesn't need to be here, it needs to be in the change log.

**Review by rlskoeser — COMMENTED** [review]
Changes look reasonable to me. Please move the line about dropping py2.7 from the readme to a change log entry for the next release.

**Comment by rlskoeser** [comment]
Saw one more small thing in the codecov report — py2.7 fallback import in `tests.py`can be removed now

File: setup.py (hunk 1 above)
Comment (kmcelwee) [reply setup.py:51]: 3.7 is included, both here: 
https://github.com/Princeton-CDH/django-pucas/blob/feature/drop-python-2/setup.py#L52

and in the test matrix:
https://github.com/Princeton-CDH/django-pucas/blob/feature/drop-python-2/.travis.yml#L8

Do you mean we should extend to Python 3.8? Let me know if I'm missing something

Comment (rlskoeser) [reply setup.py:51]: oh! I just thought it was missing because I didn't expand the diff, my mistake. No need to expand to 3.8 right now

**Review by rlskoeser — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (thatbudakguy) [reply setup.py:51]: i think we now also want to add the `'Programming Language :: Python :: 3 :: Only'` classifier, yeah?

Comment (kmcelwee) [reply setup.py:51]: Oh! yeah absolutely: #8

**Pull request merged**